package.json

```
@@ -40,7 +40,7 @@
     "grpc-web": "^1.4.2",
     "jasmine": "^3.7.0",
     "jasmine-core": "^3.7.1",
-    "typescript": "^4.9.3",
+    "typescript": "4.9.3",

```

```suggestion
    "typescript": "4.x.x",
```

would be better as we support 4.x